### ISSUE TITLE
Unable to create monitor: timezone field error

### ISSUE THREAD

**Issue by caruccio** [nauxliu/terraform-provider-cronitor#3]
Terraform v0.13.0
- Installing nauxliu/cronitor v1.0.7...
- Installed nauxliu/cronitor v1.0.7 (self-signed, key ID 59EE3DA325E65507)

When I try to create a monitor, this is what is happening:

Without field `timezone` set:

```
      + timezone = (known after apply)

...

Error: failed to create monitor, status code: 400, body: {"timezone":["This field may not be blank."]}
```

With `monitor = "America/Sao_Paulo"` set:

```
Error: "timezone": this field cannot be set
```

Any tip?

**Comment by nauxliu** [comment]
Thanks for reporting. Looks like a bug, I'll fix it later.

**Comment by nauxliu** [comment]
Fixed in d8bc573

**Comment by nauxliu** [comment]
Released v1.0.8 with the fix
